dataobj/schedule.h

```diff
@@ -169,12 +171,12 @@ class schedule_t
 	/**
 	 * Inserts a coordinate at current_stop into the schedule.
 	 */
-	bool insert(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0);
+	bool insert(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0, uint8 reverse_flag = 0);
 
 	/**
 	 * Appends a coordinate to the schedule.
 	 */
-	bool append(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0);
+	bool append(const grund_t* gr, uint8 minimum_loading = 0, uint16 waiting_time_shift = 0, uint8 coupling_point = 0, uint8 reverse_flag = 0);
```

Comment:
ditto

Reply:
reverse_flag is the upper 8 digits of stop_flags.
coupling_point and reverse_flag can be merged to the uint16 constant value.